py-polars/polars/datatypes.py

```diff
@@ -71,7 +73,12 @@ def get_args(tp: Any) -> Any:
 # act as specialisations (eg: "List" and "List[Int32]")
 PolarsDataType: TypeAlias = Union["DataTypeClass", "DataType"]
 PolarsTemporalType: TypeAlias = Union[Type["TemporalType"], "TemporalType"]
-
+OneOrMoreDataTypes: TypeAlias = Union[
+    PolarsDataType,
+    Sequence[PolarsDataType],
+    FrozenSet[PolarsDataType],
+    Set[PolarsDataType],
```

Comment:
Why not use Iterable here?

```suggestion
    Iterable[PolarsDataType],
```

Reply:
Apparently because I shouldn't type-hint at midnight 🤣 
Fixed!